src/data/faq.json

```diff
@@ -1052,35 +1054,42 @@
                                 ]
                             },
                             {
-                                "title": "Can the Corona-Warn-App read and process the QR codes from luca?",
+                                "title": "The luca app does not currently offer event registration. Can the Corona-Warn-App still read and process the QR codes from luca?",
                                 "anchor": "check_in_luca_codes",
                                 "active": false,
                                 "textblock": [
-                                    "Yes. The Corona-Warn-App can scan QR codes issued for event registration using the luca app and perform its own check-in process from the CWA."
+                                    "<b>Updated 15 July, 2022</b>",
+                                    "Since spring 2022, the event registration (check-in functionality) of the luca app is no longer possible for the time being. However, the Corona-Warn-App can scan QR codes issued for event registration using the luca app and perform its own check-in process from the CWA."
                                 ]
                             },
                             {
                                 "title": "Can luca also read and process QR codes from the Corona-Warn-App?",
                                 "anchor": "check_in_luca_cwa_codes",
                                 "active": false,
                                 "textblock": [
-                                    "No, this is not possible. The luca system transmits the contact data required for this purpose from visitors to the event or establishment in question to the health authorities as part of contact tracking, which must be health authority-compliant. However, the CWA does not provide for the transmission of contact data. Thus, QR codes are needed for this process, which are different from those required by the CWA."
+                                    "<b>Updated 15 July, 2022</b>",
+                                    "Note: Since spring 2022, the event registration (check-in functionality) of the luca app is no longer possible for the time being. Even when this functionality was still available in the luca app, it was not possible for the luca app to read and process QR codes from the CWA.<hr>",
+                                    "The luca system transmits the contact data required for this purpose from visitors to the event or establishment in question to the health authorities as part of contact tracking, which must be health authority-compliant. However, the CWA does not provide for the transmission of contact data. Thus, QR codes were needed for this process, which were different from those required by the CWA."

```

Comment:
I am quite confused by this paragraph. I suggest to delete it. It is describing unnecessary QR codes which are no longer used.

Reply:
Even if no longer used, we've decided to keep this article. The remarks above the paragraph should be a signal to users that the information below might not be up to date anymore